types/vega/vegapulseAir/distance.schema.json

```diff
@@ -0,0 +1,69 @@
+{
+  "$id": "https://akenza.io/vega/vegapulsAir/distance.schema.json",
+  "$schema": "http://json-schema.org/draft-07/schema#",
+  "type": "object",
+  "processingType": "uplink_decoder",
+  "topic": "distance",
+  "title": "Distance",
+  "properties": {
+    "distance": {
+      "$ref": "https://raw.githubusercontent.com/akenza-io/device-type-library/main/data-models/distance/schema.json#/$defs/distance/cm"
+    },
+    "fillLevel": {
+      "$ref": "https://raw.githubusercontent.com/akenza-io/device-type-library/main/data-models/distance/schema.json#/$defs/fillLevel/percent"
+    },
+    "unit": {
+      "title": "Unit",
+      "description": "Unit indicator",
```

Comment:
does this mean that the unit can change?

Reply:
Yes, but cm is the standard. I could also just not send this.